Rlog/ViewModel/Schedule/ScheduleListViewModel.swift

```diff
@@ -35,25 +37,24 @@ final class ScheduleListViewModel: ObservableObject {
 
 private extension ScheduleListViewModel {
     func fetchAllWorkDays() {
-        let result = CoreDataManager.shared.getAllWorkspaces()
         DispatchQueue.main.async { [weak self] in
             guard let self = self else { return }
-            for workspace in result {
-                self.allWorkDays.append(contentsOf: CoreDataManager.shared.getWorkdays(
-                    of: workspace,
-                    yearInt: Calendar.current.component(.year, from: Date()),
-                    monthInt: Calendar.current.component(.month, from: Date()),
-                    limit: 20)
-                )
-            }
-            self.upcomingWorkDays = self.allWorkDays.filter { self.isUpcomming(day: $0.dayInt) }
-            self.pastWorkDays = self.allWorkDays.filter { !self.isUpcomming(day: $0.dayInt) }
+            self.allWorkDays = CoreDataManager.shared.getWorkdaysByMonth(yearInt: self.year, monthInt: self.month)
+            self.upcomingWorkDays = self.allWorkDays.filter { self.isUpcomingOrNotDone(hasDone: $0.hasDone, day: $0.dayInt) }
+            self.pastWorkDays = self.allWorkDays.filter { !self.isUpcomingOrNotDone(hasDone: $0.hasDone, day: $0.dayInt) }
         }
     }
     
     // TODO: - 날짜 struct 만들기
     func isUpcomming(day: Int16) -> Bool {
-        if day >= Calendar.current.component(.day, from: Date()) {
+        if day > Calendar.current.component(.day, from: Date()) {
+            return true
+        }
+        return false
```

Comment:
```suggestion
        day > Calendar.current.component(.day, from: Date())
```

한줄로 단축이 될 것으로 보입니다 ~

Reply:
감사합니다! 수정 완료했습니다:)